tests/integration/comment.test.js

```diff
@@ -0,0 +1,103 @@
+import chaiHttp from "chai-http";
+import chai, { expect, should } from "chai";
+import app from "../../index";
+import { newArticle, newUser, newComment } from "../testData";
+import constants from "../../helpers/constants";
+
+let token;
+let slug;
+
+const { UNAUTHORIZED, CREATED, BAD_REQUEST } = constants.statusCode;
+chai.use(chaiHttp);
+
+before(done => {
```

Comment:
I can't see where you are cleaning the database after testing.

Reply:
Okay, let me work on it.